pandas/tests/io/test_parquet.py

```diff
@@ -558,11 +556,7 @@ def test_s3_roundtrip_for_dir(self, df_compat, s3_resource, pa, partition_col):
             pa,
             expected=expected_df,
             path="s3://pandas-test/parquet_dir",
-            write_kwargs={
-                "partition_cols": partition_col,
-                "compression": None,
-                "filesystem": get_s3_fs(),
```

Comment:
What happens if filesystem is provided?

Reply:
Agree + do we need to update any of the docstrings?

Reply:
It works fine, this is tested